moveit_ros/planning_interface/planning_scene_interface/src/planning_scene_interface.cpp

```diff
@@ -235,6 +251,7 @@ class PlanningSceneInterface::PlanningSceneInterfaceImpl
 private:
   ros::NodeHandle node_handle_;
   ros::ServiceClient planning_scene_service_;
+  mutable ros::ServiceClient apply_planning_scene_service_;
```

Comment:
I know ``mutable`` is for ABI, but it seems dirty and we have sonames that will force everyone to recompile anyway. I vote to remove this.